packages/ActionBar/README.md

```diff
@@ -0,0 +1,153 @@
+## Action Bar
+
+Action Bar component contains 3 individual widgets.
+
+`<Filter />` `<Sort />` and `<ColumnsArrangement />`
+
+### Installation
+
+`> npm install --save @paprika/action-bar`
+or
+`> yarn add @paprika/action-bar`
+
+### Usage
+
+#### Filter
+
+```js
+import { Filter } from "@paprika/action-bar";
+
+<Filter
+  appliedNumber={appliedNumberOfFilters}
+  columns={columns}
+  onAddFilter={onAddFilter}
+  onApply={onApply}
+  onChangeOperator={onChangeOperator}
+  operator={operator}
+>
+  <Filter.Item id={1} index={0} onChange={handleChange} onDelete={handleDelete} />
+</Filter>;
+```
+
+[More detail about props](https://github.com/acl-services/paprika/blob/aa770ab261d6364c2f14717c8edeb7d1e560a3d5/packages/ActionBar/src/components/Filter/Filter.js)
+
+You can also use the hook `useFilter`
+
+```js
+const {
+  appliedNumberOfFilters,
+  filters,
+  onAddFilter,
+  onChangeOperator,
+  onDeleteFilter,
+  onFilterChange,
+  operator,
+  onApply,
+  filteredData,
+} = useFilter({ columns, rulesByType, data });
+
+const handleDeleteFilter = filterId => () => {
+  onDeleteFilter(filterId);
+};
+const handleChangeFilter = filterId => params => {
+  onFilterChange({ ...params, id: filterId });
+};
+
+return (
+  <Filter
+    appliedNumber={appliedNumberOfFilters}
```

Comment:
I think something like 

`const props = useFilter(...)`

`<Filter {...props}  />`

is shorter and more efficient?

Reply:
nip btw, I think for documentation is fine so it is clearer what we are doing.